CONTRIBUTING.md

```diff
@@ -40,11 +40,16 @@ Install the gem:
 
 `gem install trollolo-<version>.gem`
 
-### To run rubocop
+### To run style checks
 
-To run Rubocop displaying cop names in offense messages:
+* **Ruby**: To run Rubocop displaying cop names in offense messages:
+`bundle exec rake stylecheck:ruby`
 
-`bundle exec rubocop -D`
+* **Python**: We are using pycodestyle (formerly pep8) as a tool to check Python code against some of the style conventions in PEP 8.
```

Comment:
For consistent you should explain what is Rubocop

Reply:
Done